### ISSUE TITLE
MyBot makes first post after save as Draft

### ISSUE THREAD

**Issue by cuthulino** [JN-Jones/MyBot#7]
Hello,

In our Forum we use MyBot in one Section, where people make "presentations".
After they made the Thread starting post, mybot posts the second one.
But if someone saves the thread as draft first and then activates the draft as thread,
MyBot posts as first and third. The presentation is the second post then and the thread does no longer belong to the user.

If you have to see what i mean, you can test it in our Forum.

We would give you a donation if you can fix this for us.

**Comment by AreaMaster** [comment]
I have same problem too.
Please anyone help us.

**Comment by ghost** [comment]
Yes this is a bug, I'll post it in the forum thread where hopefully it'll get some more attention.

http://community.mybb.com/thread-125876.html
